jujugui/static/gui/src/app/views/environment.js

```diff
@@ -197,12 +197,7 @@ YUI.add('juju-view-environment', function(Y) {
     */
     _handlePlusClick: function(e) {
       this.fire('changeState', {
```

Comment:
Instead of firing the event here. in the `show_environment` method in app.js, pass in the changeState method and then call it directly here.